graphql/resolvers.go

```diff
@@ -1595,7 +1595,7 @@ func (r *queryResolver) TaskAllExecutions(ctx context.Context, taskID string) ([
 }
 
 func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error) {
```

Comment:
I'm surprised that this is what's used for the dropdown; I think a future project could be a resolver that just returns the grouped names (as opposed to all of the info) since that's a lot of querying but I'm not sure if the project info is also being used in other ways.